src/components/modeler/Selection.vue

```diff
@@ -578,29 +578,54 @@ export default {
       this.overPoolStopDrag();
       this.updateSelectionBox();
       if (this.isMultiplayer) { 
-        window.ProcessMaker.EventBus.$emit('multiplayer-updateNodes', this.getProperties());
+        const changed = [];
```

Comment:
if changed is always going to be an empty array, just pass the empty array into the function instead of creating a whole new reference for the garbage collector